### PR TITLE
local-testing for 21 nodes with net delays

### PR DESCRIPTION
**Pull request by tobowers**
This creates a much bigger local docker cluster for occasional testing and also adds a command to generate configs for various sized notary groups.

It also adds a fault-detector to benchmarks which will (optionally) monitor the notary group for any signer that is disagreeing with the group.

Generally it's run like this:

```
 docker-compose -f docker-compose-localperf.yml up --build --force-recreate
```

and then

```
docker-compose -f docker-compose-localperf.yml -f docker-compose-localperf-benchmark.yml up --force-recreate --build benchmark
```

which will then run the benchmarks with the fault-detector turned on.

### CODE REVIEW

**Comment by tobowers** [comment]
I debated checking in the configs, but in the end 21 nodes is what we'll probably often want locally and it's not yet completely automated to create that.

File: cmd/generate_local_config.go [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+		SignKeysByAddress: signKeysByAddress,
+	}, nil
+}
+
+func writeLocalConfigs() error {
+	keys, err := newKeySet(localConfigNodeCount)
+	if err != nil {
+		return err
+	}
+	// humanConfigs := make([]*nodebuilder.HumanConfig, localConfigNodeCount)
```
Comment (zonotope) [inline cmd/generate_local_config.go]: this looks like an unnecessary remnant

File: cmd/generate_local_config.go [hunk 2]
```diff
@@ -0,0 +1,210 @@
...
+
+	for i := 0; i < localConfigNodeCount; i++ {
+		hc := &nodebuilder.HumanConfig{
+			NotaryGroupConfig: "../" + perfTestGroupName + ".toml",
+			PrivateKeySet: &nodebuilder.HumanPrivateKeySet{
+				SignKeyHex: hexutil.Encode(keys.SignKeys[i].Bytes()),
+				DestKeyHex: hexutil.Encode(crypto.FromECDSA(keys.EcdsaKeys[i])),
+			},
+		}
+		// humanConfigs[i] = hc
```
Comment (zonotope) [inline cmd/generate_local_config.go]: and here

File: cmd/generate_local_config.go (hunk 1 above)
Comment (tobowers) [reply cmd/generate_local_config.go]: yup oops

**Pull request merged**